core/services/ardupilot_manager/firmware_download/FirmwareDownload.py

```diff
@@ -34,11 +36,25 @@ class Platform(str, Enum):
 
     Pixhawk1 = "Pixhawk1"
     Navigator = "Navigator"
+    SITL_ARM = "SITL_arm_linux_gnueabihf"
+    SITL_X86 = "SITL_x86_64_linux_gnu"
+
+
+class FirmwareFormat(str, Enum):
+    """Valid firmware formats to download"""
+
+    APJ = "apj"
+    ELF = "ELF"
```

Comment:
Is there a reason for the capitalization here ? Can you add a comment explaining ?

Reply:
I'm trying to keep the Enums 1:1 with the manifest. They are like this there.

Reply:
So, can you add this information in the code documentation, otherwise this is not clear.

Reply:
Done.